### ISSUE TITLE
iPad: Multiple windows

### ISSUE THREAD

**Issue by samuelclay** [samuelclay/NewsBlur#1264]
I believe this is iOS 13, but I’d like to have NewsBlur open in multiple windows. (This is the feature where you tap on a iPad dock icon while the app is already open and it switches into the window switch dialog). 

Is this feasible? Are there assumptions that the app makes that can’t be resolved with multiple windows? I feel like this would require those callback pointers that you pass between pages so you can context switch and not lose your place, but I don’t know that this is absolutely necessary. I just want multiple NewsBlur windows open side by side on different feeds. A callback pointer would be necessary if it was the same feed but at different pages, which we don’t need to support.

**Comment by Dejal** [comment]
Yeah, that would be a nice enhancement. Anything is possible, but I haven't looked into it yet, so I'm not sure how difficult it'd be.  I expect that there are numerous assumptions of a single window that would have to be changed, but I can think of various ways of solving that. I'll investigate sometime.

**Comment by Dejal** [comment]
Work on this (and much more) is underway; see #817 for more info.
